x-pack/plugins/apm/public/components/app/Main/__test__/__snapshots__/Home.test.js.snap

```diff
@@ -31,9 +31,7 @@ exports[`Home component should render 1`] = `
       <SetupInstructionsLink />
     </EuiFlexItem>
   </EuiFlexGroup>
-  <EuiSpacer
-    size="l"
-  />
+  <EuiSpacer />
```

Comment:
Is `size="l"` now implicit?

Reply:
@chandlerprall Looks like https://github.com/elastic/eui/pull/1583 https://github.com/chandlerprall/eui/blob/ff76dda86abd6a7244d88cf01b0bbb23a4e398f4/src/components/spacer/spacer.tsx#L26 defaulted spacer size to `large`? Is this a breaking change?

Reply:
moved this question to the new PR